Snakemake/workflow/rules/merge_samples_vcf.smk

```diff
@@ -55,15 +49,17 @@ if len(allFiles) != 1:
             # filter files if sample list has changed, otherwise only rename
             for i in range(len(samplelist)):
                 vcf=vcflist_input[i]
+
                 ovcf=vcflist_output[i]
                 samples=samplelist[i]
                 if (i in set(track)) and (len(samples) != 0):
                     shell('bcftools view -S {samples} --force-samples {vcf} -O z -o {ovcf}')
                     shell('bcftoools index {ovcf}')
```

Comment:
There is an extra 'o' here: `bcftoools index {ovcf}`

Reply:
tooooooooools